example/cycle/test/test.js

```diff
@@ -125,31 +122,29 @@ describe('Cycles', function() {
     });
   });
 
-  // describe('searchUsers', () => {
-  //   it('should emit HTTP requests given ACTIONs, in an async fashion', (done) => {
-  //     const actionSource = {
-  //       a: actions.searchUsers('l'),
-  //       b: actions.searchUsers('lu'),
-  //       c: actions.searchUsers('luc')
-  //     };
-  //     const httpSource = {
-  //       select: () => null
-  //     }
-  //     const actionSink = {
-  //       a: {
-  //         url: `https://api.github.com/search/users?q=luc`,
-  //         category: 'query'
-  //       }
-  //     }
-  //
-  //
-  //     assertSourcesSinks({
-  //       ACTION: { '-a-b-c------|': actionSource },
-  //       HTTP:   { '-------|': httpSource },
-  //       Time:   { stream: mockTimeSource() }
-  //     }, {
-  //       HTTP:   { '--a----|': actionSink }
-  //     }, searchUsers, done);
-  //   })
-  // })
+  describe('searchUsers', () => {
+    it('should emit HTTP requests given ACTIONs, in an async fashion', (done) => {
+      const actionSource = {
+        a: actions.searchUsers('l'),
+        b: actions.searchUsers('lu'),
+        c: actions.searchUsers('luc')
+      };
+      const httpSource = {
+        select: () => null
+      }
+      const actionSink = {
```

Comment:
I think `httpSink` is a more appropriate name for this one.

Also, should there maybe be a test for making sure http *response* gets handled as well?